### ISSUE TITLE
special characters

### ISSUE THREAD

**Issue by kompowiec** [aquach/vim-mediawiki-editor#7]
If article contain "/" or eg. polish signs etc. something the margins of ASCII, plugin can't decode to:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/kompowiec2/.vim/plugged/vim-mediawiki-editor/plugin/mediawiki_editor.py", line 117, in mw_read
    vim.current.buffer[:] = s.Pages[article_name].text().split("\n")
  File "/usr/local/lib/python2.7/dist-packages/mwclient/listing.py", line 224, in __getitem__
    return self.get(name, None)
  File "/usr/local/lib/python2.7/dist-packages/mwclient/listing.py", line 245, in get
    namespace = self.guess_namespace(name)
  File "/usr/local/lib/python2.7/dist-packages/mwclient/listing.py", line 272, in guess_namespace
    if name.startswith(u'%s:' % self.site.namespaces[ns].replace(' ', '_')):
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc4 in position 8: ordinal not in range(128)

article with space can run if use sign "_" but use eg. %3F instead ó unfortunately doesn't work :(

**Comment by Thibaut120094** [comment]
Hi,

Yeah same problem here with https://fr.wikipedia.org/wiki/Wikipédia:Le_Bistro/1_décembre_2016

```
Error detected while processing function <SNR>8_MWRead:
line    2:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/thib/.vim/plugin/mediawiki_editor.py", line 117, in mw_read
    vim.current.buffer[:] = s.Pages[article_name].text().split("\n")
  File "/usr/local/lib/python2.7/dist-packages/mwclient/listing.py", line 224, i
n __getitem__
    return self.get(name, None)
  File "/usr/local/lib/python2.7/dist-packages/mwclient/listing.py", line 245, i
n get
    namespace = self.guess_namespace(name)
  File "/usr/local/lib/python2.7/dist-packages/mwclient/listing.py", line 272, i
n guess_namespace
    if name.startswith(u'%s:' % self.site.namespaces[ns].replace(' ', '_')):
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 5: ordinal
not in range(128)
```

**Comment by kompowiec** [comment]
the weirdest that on gif in which see demo plugin, he could get the rough sandbox, which in name have special character "/"! And i can't get work because of aforementioned errors :(

**Comment by aquach** [comment]
Should be fixed by #10.
